### PR TITLE
feat(specs): add `lastUpdatedAt` field to predict segments

### PR DESCRIPTION
**Pull request by bengreenbank**
## 🧭 What and Why

Adds the last updated date and time to be consistent with other data structures in Predict.

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/PRED-1281
API PR: https://github.com/algolia/predict-api/pull/97

**To be released only once the API is live with the new changes.**

### Changes included:

- Add `lastUpdatedAt` to segments
- Edit description of `updatedAt` in segments to distinguish between the two

## 🧪 Test
n/a

### CODE REVIEW

**Review by shortcuts — APPROVED** [review]
looks great :)

**Review by writeens — APPROVED** [review]

**Comment by shortcuts** [comment]
is it good to go? the pr body feels like it's not

**Comment by bengreenbank** [comment]
> is it good to go? the pr body feels like it's not

No, we plan to release the API some time next week. 

I'll comment on this PR once it's ready.

File: specs/predict/responses/Segment.yml [hunk 1]
```diff
@@ -108,6 +111,11 @@ segmentName:
   description: The name of the segment.
 
 updatedAt:
+  type: string
+  description: The date and time at which the segment was successfully updated (RFC3339).
```
Comment (francoischalifour) [inline specs/predict/responses/Segment.yml:115]: Is "successfully" needed?

```suggestion
  description: The date and time at which the segment was updated (RFC3339).
```

**Review by francoischalifour — APPROVED** [review]

File: specs/predict/responses/Segment.yml (hunk 1 above)
Comment (bengreenbank) [reply specs/predict/responses/Segment.yml]: https://github.com/algolia/api-clients-automation/pull/1431/commits/5c3cd7ec0d6c7d10c5011d701e95426f6ac8311f

Comment (bengreenbank) [reply specs/predict/responses/Segment.yml]: (Discussed on Slack) Yes, you are correct. `updatedAt` is only ever used in the response to a create or update segment request (not used by the pipeline).

**Review by shortcuts — APPROVED** [review]

**Pull request merged**